scripts-available/CDB_Organizations.sql

```diff
@@ -25,6 +25,57 @@ BEGIN
 END
 $$ LANGUAGE PLPGSQL VOLATILE;
 
+-------------------------------------------------------------------------------
+-- Administrator
+-------------------------------------------------------------------------------
+CREATE OR REPLACE
+FUNCTION cartodb._CDB_Organization_Admin_Role_Name()
+    RETURNS TEXT
+AS 'SELECT ''cdb_org_admin''::text || ''_'' || md5(current_database());'
```

Comment:
This comes from the discussion with @rochoa about identifier characters and lengths. Since current_database is a postgresql identifier prepending `cdb_org_admin_` might exceed maximum valid length.